source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow-and-button.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Heading with arrow and button
+ * Slug: wporg-parent-2021/heading-with-arrow-and-button
+ * Categories: header
+ */
+
+?>
+<!-- wp:group {"align":"full","backgroundColor":"blueberry-1"} -->
+<div class="wp-block-group alignfull has-blueberry-1-background-color has-background">
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"97px","bottom":"97px","right":"80px","left":"80px"}}},"className":"heading-with-arrow","layout":{"type":"flex","orientation":"vertical","justifyContent":"center","flexWrap":"nowrap"}} -->
+	<div class="wp-block-group alignfull heading-with-arrow" style="padding-top:97px;padding-right:80px;padding-bottom:97px;padding-left:80px">
+		<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"inherit":false}} -->
+		<div class="wp-block-group">
+			<!-- wp:group {"style":{"spacing":{"padding":{"bottom":"67px"}}},"className":"heading-with-arrow__row","layout":{"type":"flex","flexWrap":"nowrap","orientation":"horizontal","justifyContent":"center","verticalAlignment":"top"}} -->
+			<div class="wp-block-group heading-with-arrow__row" style="padding-bottom:67px">
+				<!-- wp:group {"layout":{"contentSize":"760px","inherit":false}} -->
+				<div class="wp-block-group">
```

Comment:
Why do you need this many group blocks? Could it be simplified?

Reply:
Originally I was thinking to make the patterns able to be customized easily with GB UI tools from users. So I tried to make the pattern with GB UI tools as much as possible. In this example, basically every aspect of the UI layout can be tweaked through GB UI tools. But from other comments, I guess I know what we're trying to make here. I'll adjust it!